docs/pages/admin-guides/management/guides/scim-sailpoint.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: SCIM Sailpoint Integration
+description: How to manage Access List membership via SCIM SailPoint integration
+
+---
+
+The SCIM integration between **SailPoint** and **Teleport** enables automated
+synchronizing SailPoint group memberships with
+Teleport Access Lists. This integration supports centralized identity governance in
+SailPoint while Teleport enforces fine-grained access controls.
+
+User permissions in Teleport are defined through Access Lists. While role definitions
+live in Teleport, **group membership is dynamically managed** by SailPoint SCIM group
+assignments. This ensures users have up-to-date access aligned with organizational
+policies.
+
+## Prerequisites
+- Teleport Enterprise v(=teleport.version=)
+- Teleport Terraform provider v(=teleport.version=)
+- A running Teleport cluster with SSO connect (e.g., Okta SAML enabled
+- Access to SailPoint IdentityNow or IdentityIQ
+
+
+
+### How it works
+
+The SCIM integration uses a **1:1 mapping** between SailPoint groups and Teleport
+Access Lists:
+
+- Each SailPoint group’s `displayName` must match the `spec.title` of a Teleport
+  Access List.
+- Access Lists must be created in advance in Teleport and set to `type: "scim"`.
+- Role assignments are handled in Teleport, while group membership is managed by
+  SailPoint.
+
+
+### Step 1/3. Configure SCIM Integration
```

Comment:
Some "Step..." headings use the H2 level while some use H3. We should use H2 for all of them.